docs/tutorial/part_1.rst

```diff
@@ -0,0 +1,331 @@
+Writing your first Channels app, part 1
+=======================================
+
+In this tutorial we will build a simple chat server. It will have two pages:
+
+* An index view that lets you type the name of a chat room to join.
+* A room view that lets you see messages posted in a particular chat room.
+
+The room view will use a WebSocket to communicate with the Django server and
+listen for any messages that are posted.
+
+We assume that you are familar with basic concepts for building a Django site.
+If not we recommend you complete `the Django tutorial`_ first and then come back 
+to this tutorial.
+
+We assume that you have `Django installed`_ already. You can tell Django is
+installed and which version by running the following command in a shell prompt
+(indicated by the ``$`` prefix)::
+
+    $ python3 -m django --version
+
+We also assume that you have :doc:`Channels installed </installation>` already. You can tell
+Channels is installed by running the following command::
+
+    $ python3 -c 'import channels; print(channels.__version__)'
+
+This tutorial is written for Channels 2.0, which supports Python 3.5+ and Django
+1.11+. If the Channels version does not match, you can refer to the tutorial for
+your version of Channels by using the version switcher at the bottom left corner
+of this page, or update Channels to the newest version.
+
+This tutorial also uses Docker to install and run Redis. We use Redis as the
+backing store for the channel layer, which is an optional component of the
+Channels library that we use in the tutorial. `Install Docker`_ from its
+official website.
+
+.. _the Django tutorial: https://docs.djangoproject.com/en/2.0/intro/tutorial01/
+.. _Django installed: https://docs.djangoproject.com/en/2.0/intro/install/
+
+.. _Install Docker: https://www.docker.com/get-docker
+
+Creating a project
+------------------
+
+If you don’t already have a Django project, you will need to create one.
+
+From the command line, **cd** into a directory where you’d like to store your
```

Comment:
I would use fixed-width text (so ``` ``like this`` ```) rather than bold for this and other command-line things, like filenames and the like below.